mozdownload/scraper.py

```diff
@@ -384,6 +388,10 @@ def __init__(self, branch='mozilla-central', build_id=None, date=None,
     def get_build_info(self):
         """Defines additional build information"""
 
+        # Find branch by application
+        if self.branch is None:
+            self.branch = DEFAULT_BRANCHES[self.application]
```

Comment:
It is but in the base scraper class: https://github.com/TLemur/mozdownload/blob/issue_247/mozdownload/scraper.py#L129